### PR TITLE
Update eslint packages

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: src/popup/App.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
         @click.self="showSidebar = false"
         data-cy="menu-overlay"
       >
-        <SidebarMenu @closeMenu="showSidebar = false" />
+        <SidebarMenu @close="showSidebar = false" />
```
Comment (mradkov) [inline src/popup/App.vue:22]: Should we change this naming in the scope of this PR?

File: src/mixins/removeAccount.js [hunk 2]
```diff
@@ -13,11 +13,11 @@ export default {
         })
         .catch(() => false);
       if (remove) {
-        this.$emit('closeMenu');
+        this.$emit('close');
```
Comment (mradkov) [inline src/mixins/removeAccount.js:16]: Should we change this naming in the scope of this PR?

File: src/popup/router/components/SidebarMenu.vue [hunk 3]
```diff
@@ -115,7 +115,7 @@ export default {
       if (target.tagName === 'A') this.closeMenu();
     },
     closeMenu() {
-      this.$emit('closeMenu');
+      this.$emit('close');
```
Comment (mradkov) [inline src/popup/router/components/SidebarMenu.vue:118]: Should we change this naming in the scope of this PR?

File: src/popup/router/pages/Tip.vue [hunk 4]
```diff
@@ -58,13 +58,13 @@
...
-              !url ||
-              urlStatus === 'blacklisted' ||
-              note.length > 280
+            amountError ||
+            noteError ||
+            !minCallFee ||
+            !validUrl ||
+            !url ||
+            urlStatus === 'blacklisted' ||
+            note.length > 280
```
Comment (mradkov) [inline src/popup/router/pages/Tip.vue:67]: Not specific for this PR, but we should document somewhere this behavior and the requirements for tip note to be less than 280 characters length.

File: src/popup/utils/phishing-detect.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-const phishingCheckUrl = async url => {
+const phishingCheckUrl = async (url) => {
   try {
     const res = await fetch(`https://etherscamdb.info/api/check/${url}`);
```
Comment (mradkov) [inline src/popup/utils/phishing-detect.js:3]: Should we maybe extract this somewhere in the constants?

File: src/store/getters.js [hunk 6]
```diff
@@ -30,9 +30,9 @@ export default {
   activeNetwork({ current: { network } }, { networks }) {
     return networks[network];
   },
-  getProfileImage: (_, { activeNetwork }) => address =>
+  getProfileImage: (_, { activeNetwork }) => (address) =>
     `${activeNetwork.backendUrl}/profile/image/${address}`,
-  getAvatar: () => address => `https://avatars.z52da5wt.xyz/${address}`,
+  getAvatar: () => (address) => `https://avatars.z52da5wt.xyz/${address}`,
```
Comment (mradkov) [inline src/store/getters.js:35]: Can we extract the avatars service URL in a constant in the config?

File: src/popup/App.vue (hunk 1 above)
Comment (davidyuk) [reply src/popup/App.vue:22]: The original eslint error was:
> /superhero-wallet/src/popup/router/components/SidebarMenu.vue
  118:18  error  Custom event name 'closeMenu' must be kebab-case  vue/custom-event-name-casing

So, I have to update it and at the same time, I thought that the event emitted inside the component shouldn't contain the component name. Then I decided to don't edit this twice 🙃

**Review by CedrikNikita — APPROVED** [review]

**Comment by mradkov** [comment]
@davidyuk can you rebase this PR with latest develop. Thanks

**Pull request merged**